docs/sources/flow/reference/components/loki.rules.kubernetes.md

```diff
@@ -0,0 +1,259 @@
+---
+title: loki.rules.kubernetes
+labels:
+  stage: beta
+---
+
+# loki.rules.kubernetes
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`loki.rules.kubernetes` discovers `PrometheusRule` Kubernetes resources and
+loads them into a Loki instance.
+
+* You can specify multiple `loki.rules.kubernetes` components by giving them different labels.
+* [Kubernetes label selectors][] can be used to limit the `Namespace` and
+  `PrometheusRule` resources considered during reconciliation.
+* Compatible with the Ruler APIs of Grafana Loki, Grafana Cloud, and Grafana Enterprise Metrics.
+* Compatible with the `PrometheusRule` CRD from the [prometheus-operator][].
+* This component accesses the Kubernetes REST API from [within a Pod][].
+
+{{% admonition type="note" %}}
+This component requires [Role-based access control (RBAC)][] to be set up
+in Kubernetes for {{< param "PRODUCT_ROOT_NAME" >}} to access it via the Kubernetes REST API.
+{{% /admonition %}}
```

Comment:
```suggestion

Role-based access control (RBAC)]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
{{% /admonition %}}
```

We need to move the link definition inside the admonition shortcode for the rendering to work correctly.